Samples/Mapsui.Samples.Common/Maps/Projection/MultiPolygonProjectionSample.cs

```
@@ -41,7 +41,8 @@ public static Map CreateMap()
             };
             map.Layers.Add(OpenStreetMap.CreateTileLayer());
             map.Layers.Add(geometryLayer);
-            map.Home = n => n.NavigateTo(extent);
+            if (extent != null)
+                map.Home = n => n.NavigateTo(extent);
```

Perhaps we should throw an exception right after extent is created if it is null.

---

I don't know how the sample app behaves in that case. I think complete crash. That is not what we want.